Dockerfile

```diff
@@ -1,18 +1,17 @@
-FROM registry.fedoraproject.org/fedora:38-x86_64
+FROM registry.fedoraproject.org/fedora:39-x86_64
 
 LABEL \
     name="Freshmaker application" \
     vendor="Freshmaker developers" \
-    license="GPLv2+" 
-
+    license="GPLv2+"
 
 # Use Copr repo for python3-rhmsg package
-RUN dnf install -y 'dnf-command(copr)' && dnf copr enable -y qwan/python-rhmsg
+RUN dnf install -y 'dnf-command(copr)' && dnf copr enable -y qwan/python-rhmsg && dnf copr enable -y mfoganho/python3-qpid-proton
```

Comment:
so what about rhmsg? I thought we were supposed to compile a new version of it for f39

Reply:
Good point! When I started working on the task, a build for f39 was already done:
https://copr.fedorainfracloud.org/coprs/qwan/python-rhmsg/build/6583645/

I'm not sure if it was because of some automation, or because Qixiang was a step ahead, but it was certainly a pleasant surprise! 😂

Reply:
Awesome!